release/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id "com.palantir.docker" version "0.25.0"
+}
+
+dependencies {
+    compile project(':situp-core')
+}
+
+task endToEndTests {
+    //dependsOn(':situp-api:test') TODO add E2E and enable when ready
+}
+
+task benchmarkTests {
+    //dependsOn(':situp-api:test') TODO add benchmark test and enable
+}
+
+task buildCore {
+    dependsOn ':situp-core:build'
+}
+
+task dockerPrerequisites {
+    dependsOn 'buildCore'
+    dependsOn 'endToEndTests'
+    dependsOn 'benchmarkTests'
+}
+
+docker {
+    name "odfe-pipes/${project.rootProject.name}:${project.version}"
+    tag "${project.name}", "${project.version}" //update tag when ready to push
+    files project(':situp-core').jar.archivePath
+    buildArgs(['JAR_FILE': project(':situp-core').jar.archiveName,
+               'CONFIG_FILEPATH': '/usr/share/situp/odfe-pipes.yml'])
```

Comment:
QUES: is it expected user put config file in this exact location? Shall we allow custom file path?

Reply:
Good Ques - it is the default location - user should be able to pass in the different location if he intends to - I will add that as part of next PR.

Reply:
Users are expected to use the docker image, they will not building a new image. Why would users want custom path?

Reply:
@kowshikn Maybe I misunderstood this line. So is `CONFIG_FILEPATH` just a **default** location looking for the config file when user runs the image?